cmd/snap/cmd_paths_test.go

```diff
@@ -70,4 +71,20 @@ func (s *SnapSuite) TestPathsArch(c *C) {
 		"SNAPD_BIN=/var/lib/snapd/snap/bin\n"+
 		"SNAPD_LIBEXEC=/usr/lib/snapd\n")
 	c.Assert(s.Stderr(), Equals, "")
+
+	s.ResetStdStreams()
+
+	// new contents, as set by filesystem-2018.12-1
+	restore = release.MockReleaseInfo(&release.OS{ID: "archlinux"})
+	defer restore()
+	defer dirs.SetRootDir("/")
```

Comment:
Hmm? So both defer and call here?

Reply:
uhh, right, i'll force push

Reply:
+1. Present also in the existing code for the old arch, above.